### ISSUE TITLE
rawCmd documentation 

### ISSUE THREAD

**Issue by EvaldoNeto** [luqasz/librouteros#78]
I am not sure if i am using it wrong or if it is broken. Below it is what i tried to do. Everything seems to run smoothly, but the backup is not generated.

`
api = connect(
            username='admin',
            password='my_password',
            host='my_ip',
            timeout=20,
          )
api.rawCmd('/system/backup/save')
`

So, if it is working properly, what is wrong?

**Comment by luqasz** [comment]
`api.rawCmd` return a generator which must be consumed.

**Comment by EvaldoNeto** [comment]
Thanks for pointing that out. I was able to consume it, but I could not add parameters. I tried the below code and some variants like "?name=mybacku", "?=name=mybackup", "name=\"my_backup\""...
`
for i in  api.rawCmd('/system/backup/save', 'name=my_backup'):
        pass
`

I also tried to follow an old example in the documentation with no success

`api.rawCmd('/ip/address/print', '?=address=1.1.1.1')`

So... how should the arguments be passed?
